package.json

```diff
@@ -23,15 +25,31 @@
   "license": "MIT",
   "devDependencies": {
     "browserify": "^11.2.0",
+    "eslint": "^6.8.0",
+    "eslint-config-prettier": "^6.10.1",
+    "eslint-plugin-prettier": "^3.1.3",
+    "husky": "^4.2.5",
     "karma": "^0.13.10",
     "karma-browserify": "^4.4.0",
     "karma-chrome-launcher": "^2.2.0",
     "karma-firefox-launcher": "^1.0.1",
     "karma-safari-launcher": "^1.0.0",
     "karma-tap": "^1.0.3",
     "karma-tape-reporter": "^1.0.3",
+    "lint-staged": "^10.1.3",
+    "prettier": "2.0.4",
     "tape": "^4.2.1"
   },
   "dependencies": {},
-  "repository": "https://github.com/chmanie/o9n.git"
+  "repository": "https://github.com/chmanie/o9n.git",
+	"husky": {
+		"hooks": {
+			"pre-commit": "lint-staged"
+		}
+	},
+	"lint-staged": {
+		"*.{js}": [
+			"npm run format"
```

Comment:
I think this should actually just run `eslint --fix`, otherwise lint-staged will run eslint on all files (well, it doesn't make a big difference in this repo, just for good measure, I guess :) )

See https://github.com/okonet/lint-staged#what-commands-are-supported

Reply:
I have no problem moving this over. Like you'd mentioned, this shouldn't impact anything right now, but changing it for later might be ideal